libraries/libcore/inc/status.h

```diff
@@ -52,6 +52,7 @@ typedef enum {
   STATUS_CODE_OUT_OF_RANGE,
   STATUS_CODE_UNIMPLEMENTED,
   STATUS_CODE_UNINITIALIZED,
+  STATUS_CODE_CRC_MISMATCH,
```

Comment:
Should this be a status? I feel like it is not very generic which is what I want the status codes to be. I'd rather this be `STATUS_CODE_INTERNAL_ERROR` or similar and then use the message to provide a more verbose description.